packages/@aws-cdk/aws-apigatewayv2/README.md

```diff
@@ -79,6 +79,11 @@ httpApi.addRoutes({
   methods: [ HttpMethod.ANY ],
   integration: booksDefaultIntegration,
 });
+
+new CfnOutput(stack, 'my-api-url', {
+  exportName: 'ApiUrl',
+  value: httpApi.apiEndpoint,
+});
```

Comment:
Instead of this, add a statement to the paragraph above to the effect "The URL to the endpoint can be retrieved via the `apiEndpoint` attribute".